.github/workflows/pytest-core-mpi.yml

```diff
@@ -39,15 +39,23 @@ jobs:
       run: |
         pytest --cov --cov-config=.coveragerc --cov-report=xml -m parallel tests/
 
-    - name: Test mpi notebook
+    - name: Test notebooks
       run : |
         # Currently not tested due to issue #859
         # ipcluster start --profile=mpi -n 4 --daemon
         # py.test --nbval examples/mpi
         # ipcluster stop --profile=mpi
 
-    - name: Test mpi fwi
-      run: mpirun -n 4 python examples/seismic/inversion/fwi.py
+    - name: Test seismic examples
```

Comment:
We should make this (and the fwi example) a separate job so that it 'parallelizes'. We can then always add an additional runner is required.